modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/NumericDDMFormFieldUtil.java

```diff
@@ -12,38 +12,27 @@
  * details.
  */
 
-package com.liferay.dynamic.data.mapping.form.field.type.internal.numeric;
+package com.liferay.dynamic.data.mapping.util;
 
 import java.text.DecimalFormat;
 
 import java.util.Locale;
-import java.util.Map;
-import java.util.concurrent.ConcurrentHashMap;
 
 /**
  * @author Rafael Praxedes
- * @deprecated As of Cavanaugh (7.4.x)
+ * @author Guilherme Camacho
  */
-@Deprecated
 public class NumericDDMFormFieldUtil {
 
 	public static DecimalFormat getNumberFormat(Locale locale) {
```

Comment:
Could we call this `getDecimalFormat`? I think it makes more sense since the return type is `DecimalFormat`.

Reply:
> Could we call this `getDecimalFormat`? I think it makes more sense since the return type is `DecimalFormat`.

Done